### PR TITLE
Inject remember_me services into LoginManager (support symfony 2.3 and higher)

### PR DESCRIPTION
**Pull request by sirian**
Fix Remember me problem with Symfony 2.3.x (https://github.com/FriendsOfSymfony/FOSUserBundle/issues/1221, https://github.com/FriendsOfSymfony/FOSUserBundle/issues/747).
It would be better if remember_me services has some tag (link issue https://github.com/symfony/symfony/issues/3137)

### CODE REVIEW

**Comment by smarques** [comment]
+1

**Comment by gondo** [comment]
any reasons why this is not merged yet?

File: DependencyInjection/Compiler/InjectRememberMeServicesPass.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\UserBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\Config\Resource\FileResource;
```
Comment (gondo) [inline DependencyInjection/Compiler/InjectRememberMeServicesPass.php]: `FileResource` is not needed as it is not used

File: DependencyInjection/Compiler/InjectRememberMeServicesPass.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+
+        $rememberMeServices = array();
+        foreach ($container->getDefinitions() as $id => $definition) {
+            if (0 !== strpos($id, 'security.authentication.rememberme.services.')) {
```
Comment (gondo) [inline DependencyInjection/Compiler/InjectRememberMeServicesPass.php]: wouldn't it possible/easier to get service based on configured firewall in `fos_user.firewall_name` ?

**Comment by artggd** [comment]
Any update on this?

**Comment by danieledangeli** [comment]
+5 (including all my friends who need it) :)

**Comment by XWB** [comment]
Maybe @sirian is willing to update this PR with the latest master branch?

**Comment by sirian** [comment]
@XWB and wait another 2 years?)

**Comment by XWB** [comment]
@sirian I was not a FOS member back then :)

**Comment by XWB** [comment]
https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2257

**Pull request closed without merging**